requirements/debug-dist.txt

```diff
@@ -0,0 +1,17 @@
+-r ../requirements.txt
+
+# Requirements that are used locally only for a good developer experience
```

Comment:
Just to mention the difference, missing two removed packages. I am OK removing `django-sslserver` but why removing `django-silk`? I will add it back in my update.

```
django-sslserver==0.19
django-silk==2.0.0
```